### ISSUE TITLE
Create a training, validation, test split utility function

### ISSUE THREAD

**Issue by bwile** [danielhwile/fantasyfootball#2]
@danielhwile For reference:

For any ML model, we'll want to split the data into three subsets:
1. Training Data: 2015-2018
2. Validation Data: 2019 First 8 Weeks
3. Test Data 2019 Last 8 Weeks

We'll use the validation data set to evaluate the model the performs the best. The test set will likely have slightly worse performance than the validation set, but it will represent the realistic performance of the model
